.circleci/config.yml

```diff
@@ -232,6 +232,25 @@ jobs:
           os: windows
           gotestsum: "gotestsum.exe"
 
+  test-licenses:
+    executor: telegraf-ci
+    steps:
+      - checkout
+      - restore_cache:
+          key: go-mod-v1-{{ checksum "go.sum" }}
+      - check-changed-files-or-halt
+      - run: 'make build_tools'
+      - run: 'sh ./tools/license_checker/license_checker -whitelist ./tools/license_checker/data/whitelist'
+      - save_cache:
+          name: 'go module cache'
+          key: go-mod-v1-{{ checksum "go.sum" }}
+          paths:
+            - '/go/pkg/mod'
+      - persist_to_workspace:
+          root: '/go'
+          paths:
+            - '*'
```

Comment:
Not sure if this is necessary, I assume nothing was added to the `/go` dir that would need to be saved.

Reply:
Removed. Thanks for catching this!